azure-pipelines.yml

```diff
@@ -209,10 +212,16 @@ stages:
     - job: Windows_Installer
       displayName: Create Installer
       pool:
-        vmImage: 'windows-2019'
+        vmImage: ${{ variables.windowsImage }}
       steps:
       - checkout: self
         fetchDepth: 1
+      - pwsh: |
+          Invoke-WebRequest https://files.jrsoftware.org/is/6/innosetup-6.2.0.exe -OutFile innosetup.exe
```

Comment:
Do we want this in the build script instead of the pipeline so we can still build installer locally easy?

Reply:
Good idea, now it's `./build.sh --installer` to build the installer once everything else is done